lib/v2/ciweimao/chapter.js

```diff
@@ -0,0 +1,57 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+module.exports = async (ctx) => {
+    const { id } = ctx.params;
+    const limit = Number.parseInt(ctx.query.limit) || 10;
+
+    const baseUrl = 'https://wap.ciweimao.com';
+    const chapterUrl = 'https://mip.ciweimao.com';
+
+    const { data: response } = await got(`${baseUrl}/book/${id}`);
+    const $ = cheerio.load(response);
+
+    const firstChapterUrl = $('ul.catalogue-list li a').attr('href');
+    const firstChapterId = firstChapterUrl.substring(firstChapterUrl.lastIndexOf('/') + 1);
+
+    const { data: chapter } = await got(`${chapterUrl}/chapter/${id}/${firstChapterId}`);
+    const $c = cheerio.load(chapter);
+
+    const list = $c('ul.book-chapter li a')
+        .slice(-limit)
+        .toArray()
+        .map((item) => {
+            item = $(item);
+            return {
+                title: item.find('h3').text(),
+                pubDate: timezone(parseDate(item.find('p').text().replace('发布于 ', '')), +8),
+                link: String(item.attr('href')),
```

Comment:
[`attr()`](https://cheerio.js.org/docs/api/classes/Cheerio#attr) will return string. 
No need to recast the same type.
docs: https://docs.rsshub.app/joinus/advanced/script-standard